content/_data/changelogs/weekly.yml

```
@@ -22089,6 +22089,54 @@
   # pull: 8699 (PR title: Update Yarn to v4.0.2)
   # pull: 8703 (PR title: Update dependency lit to v3.1.0)
 
+  - version: '2.434'
+    date: 2023-11-28
+    changes:
+      - type: rfe
+        category: rfe
+        pull: 8705
+        authors:
+          - janfaracik
+          - NotMyFault
+        pr_title: Use symbols for build status
+        message: |-
+          Refine build status icons.
+      - type: rfe
+        category: rfe
+        pull: 8694
+        authors:
+          - Santhosh-CloudB
+        pr_title: Added validation of FIPS password length
+        message: |-
+          Fips mode now requires a minimum of 14 characters for a password.
```

An acronym, all caps.
```suggestion
          FIPS mode now requires a minimum of 14 characters for a password.
```